components/Sidebar.tsx

```diff
@@ -169,7 +169,7 @@ const Sidebar = ({
   return (
     <aside
       className={`fixed left-0 bottom-0 ${
-        state.hasBanner ? "top-32" : "top-20"
+        state.hasBanner ? "top-32" : "top-16"
```

Comment:
I think `top-20` looks nicer

this is `top-16`:
![image](https://user-images.githubusercontent.com/115862/236131880-9ada3335-11ba-49b6-a70f-d7428dab0db5.png)

Reply:
What? That's quite weird, on my screen there was a gap, that's why I had to reduce it, I'll double check

Reply:
Not sure what happened, I just reversed the change and it was ok